doc/distributions/nc_chi_squared.qbk

```diff
@@ -72,14 +72,14 @@ for different values of [lambda]:
 
       non_central_chi_squared_distribution(RealType v, RealType lambda);
 
-Constructs a Chi-Squared distribution with /v/ degrees of freedom
+Constructs a Chi-Squared distribution with [nu] degrees of freedom
 and non-centrality parameter /lambda/.
 
 Requires v > 0 and lambda >= 0, otherwise calls __domain_error.
 
       RealType degrees_of_freedom()const;
 
-Returns the parameter /v/ from which this object was constructed.
+Returns the parameter v from which this object was constructed.
```

Comment:
Should this be a [nu] as well?

Reply:
ya but 
I thought if someone copy-paste the same into the code  to get the value
I don't know exactly you suggest

Reply:
ya i missed that one

Reply:
@ambreshbiradar9 : Yes, edit again in a new PR.